ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection/channel/index.js

```diff
@@ -24,7 +25,14 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       try {
         asciicast = await channelRecording.getAsciicast();
       } catch (e) {
-        // no op
+        // Alert user of any error messages.
+        e.errors?.forEach((error) => {
+          this.flashMessages.danger(error.detail, {
+            notificationType: 'error',
+            sticky: true,
+            dismiss: (flash) => flash.destroyMessage(),
```

Comment:
after message dismissal, should we redirect them to the list view?  🤔

Reply:
good point, though I can't think of any place where we redirect on message dismissal, im thinking that could be confusing to the user. Also i'm not sure if dismissal of an alert should have any side effects.

Reply:
yeah we prob won't need that.